### DISCUSSION TITLE
how to share variable across collections, if possible

### DISCUSSION THREAD

**Discussion by sainathdk2022** [usebruno/bruno#1656] [Q&A]
Greetings of the day!.

I have a collection where I get my JWT. I then want to set it to an environment variable so that I can use it across my other related collections. Is this possible to do in the free version of Bruno?

Also would this be available in the personal paid version? Thank you and keep up the great work, with Bruno. 

Best Regards,
Sainath

**Comment by jeremygau** [comment]
I would like to be able to do that too. Has anyone found a solution ?
